web/viewer-geckoview.css

```
@@ -272,9 +272,10 @@ dialog .toolbarField {
 dialog .separator {
   display: block;
   margin: 4px 0;
-  height: 1px;
+  height: 0;
   width: 100%;
-  background-color: var(--separator-color);
+  border-top: 1px solid var(--separator-color);
+  border-bottom: 0;
```

Can we please make this consistent with all the other cases?
```suggestion
  border-bottom: none;
```